src/ui/components/Shared/TracksListHeaderCell.react.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import Icon from 'react-fontawesome';
+
+import LibraryActions from '../../actions/LibraryActions';
+
+
+class TracksListHeaderCell extends React.Component {
+  static propTypes = {
+    className: PropTypes.string,
+    sortBy: PropTypes.string,
+    title: PropTypes.string,
+    onClick: PropTypes.func,
+    icon: PropTypes.string,
+  }
+
+  constructor(props) {
+    super(props);
+    this.sort = this.sort.bind(this);
+  }
+
+  sort() {
+    LibraryActions.sort(this.props.sortBy);
+  }
+
+  getCellContent(props) {
+    return (
+      <React.Fragment>
+        <div className='col-name'>
+          {props.title}
+        </div>
+        {props.icon &&
+          <div className='col-icon'>
+            <Icon name={props.icon} />
+          </div>
+        }
+      </React.Fragment>
+    );
+  }
+
+
+  render() {
+    const { sortBy, className } = this.props;
+    const classes = classnames('track-cell-header', className, {
+      sort: sortBy,
+    });
+
+    if (sortBy) {
```

Comment:
do you think it might be better / more clear to always render the button, but have it in "unsorted" state by default instead of rendering it conditionally?

Reply:
I went with the semantic approach here, a disabled button does not really make a lot of sense I think.